admin/react/assets/src/dashboard-app/pages/welcome/Welcome.js

```diff
@@ -0,0 +1,118 @@
+import { useState } from "react";
+import CustomFontGrid from "./CustomFontGrid";
+import CustomFontList from "./CustomFontList";
+import SearchBar from "./SearchBar";
+import { Link } from "react-router-dom";
+import { __ } from "@wordpress/i18n";
+
+const Welcome = () => {
+	const [activeView, setActiveView] = useState("list");
+	const [searchResults, setSearchResults] = useState(null);
+	const [loading, setLoading] = useState(false);
+	const toggleView = (value) => {
+		setActiveView(value);
+	};
+	return (
+		<div className="">
+			<div className="bg-white border-b border-slate-200">
+				<div className="max-w-3xl mx-auto px-3 sm:px-6 lg:max-w-full">
+					<div className="relative py-7">
+						<div className="flex flex-col lg:flex-row justify-between items-start">
+							<div>
+								<h2 className="text-[1.625rem] font-semibold mb-3.5">
+									{ __( "Custom Fonts", "custom-fonts" ) }
+								</h2>
+								<p className="text-xs">
+									{ __( "Manage fonts that are used on your website.", "custom-fonts" ) }
+								</p>
+							</div>
+							<div className="relative tablet:mt-4">
+								<Link
+									to={{
+										pathname: "admin.php",
+										search: `?page=bsf-custom-fonts&path=add-fonts`,
+									}}
+									className="flex text-sm text-white hover:text-white bg-primary px-3 py-2"
+								>
+									{ __("Add New Font", "custom-fonts") }
+								</Link>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+			<div className="max-w-3xl mx-auto px-3 sm:px-6 lg:max-w-full">
+				{/* Search Custom Font */}
+				<SearchBar setSearchResults={ setSearchResults } setLoading={ setLoading }/>
+				{/* Font Counter and View toggle */}
+				<div className="flex justify-between items-center my-6">
+					<div className="text-base"> { searchResults ? searchResults.found_posts : bsf_custom_fonts_admin.fontPostCount } { __(" font families", "custom-fonts") } </div>
+					<div className="flex justify-end gap-x-7">
+						<div
+							className="cursor-pointer"
+							onClick={() => toggleView("list")}
+						>
+							<svg
```

Comment:
Same here @imnavanath regarding svg reuse

Reply:
Added separate task in backlog for this - https://brainstormforce.atlassian.net/browse/CFP-38